docs/releases/4.2.12.txt

```diff
@@ -0,0 +1,14 @@
+===========================
+Django 4.2.12 release notes
+===========================
+
+*Expected May 6, 2024*
+
+Django 4.2.12 fixes a bug in 4.2.11.
+
+Bugfixes
+========
+
+* Fixed a crash in Django 4.2 when validating email max line lengths with
+  content decoded using the ``surrogateescape`` error handling scheme,
+  particularly in Python versions 3.11.9, 3.12.3, and 3.13 (:ticket:`35361`).
```

Comment:
```suggestion
  particularly in Python versions 3.11.9+ and 3.12.3+ (:ticket:`35361`).
```
Would have a similar comment for the one below as we don't support Python 3.13 currently and won't support it for 4.2 and 5.0, so we probably shouldn't mention it in the release notes here.
Python 3.12.3 is still not released yet although it was scheduled for yesterday, but I am expecting it to be released by May 6th. Not sure if we want to merge mentioning it before it is released or remove it 🤔

Reply:
Thank you, you are completely right about 3.13, we shouldn't mention it. I would mention 3.12.3 though, changing and pushing!